tests/functional/conftest.py

```diff
@@ -40,7 +40,7 @@ def do_submit_known_email_address(email_address):
 
 
 @pytest.fixture
-def click_change_email_link_on_dept_confirm(browser):
-    def do_click_change_email_link_on_dept_confirm():
-        browser.find_by_css('#confirm-dept a').click()
-    return do_click_change_email_link_on_dept_confirm
+def click_change_email(browser):
+    def do_click_change_email():
+        browser.find_by_css('#change_email').click()
+    return do_click_change_email
```

Comment:
This one might be unnecessary now - `browser.find_by_css('#change_email').click()` is pretty self-explanatory.
Just pass `browser` into the `setup_page` method and execute this directly.